imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -129,7 +129,17 @@ authenticated by supplying the access token in the `Authorization` header.
     {% /code %}
 
 3.  [Submit the signed challenge along with the
-    signature](/api-reference/login) to get the access token.
+    signature](/api-reference/login) to get the access token and a refresh token.
+    The refresh token is used to obtain a new access token when the current
+    access token expires, ensuring continuous access without requiring the user to log in again.
 4.  The access token should be used for subsequent requests for protected
     resources from the Immersve API by supplying it in the `Authorization`
     header.
+
+## Use Refresh Tokens
+
+The refresh token is a long-lived token that can be used to obtain a new access token without requiring the user to re-authenticate.
+
+Managing Refresh Token
+- Before making a request, check the expiry of the auth token. If the token is expired, use the refresh token to obtain a new auth token.
+- To exchange the refresh token you received during authentication for a new access token, call the [Auth Token Exchange API](/api-reference/auth-token-exchange). A new access token will be issued.
```

Comment:
This looks correct, user should check if their auth token is expired before every request they make, and if it is expired they need to use the refresh token to obtain a new access token first.

@magsharon can we please change 'auth token' to 'access token' (this is what the token is called in the auth-token-exchange response).

The second sentence might also read better something like:
> To obtain a new access token, call the [Auth Token Exchange API](/api-reference/auth-token-exchange) using the refresh token you received during authentication. This will return a new access token and refresh token to be exchanged when the new access token expires.

Reply:
Thank you @NicBathgate, adjustmants are done ✅